build_with_proto.sh

```diff
@@ -14,10 +14,7 @@ npm install
 npm run build -- --config grpc/src/main/resources/assets/pt/webpack.config.js
 
 # Bundle files needed for non-PT GUI
-cd web && npm install && npm run bundle && cd ..
+cd web && npm install && npm run bundle
+cd ..
 
-mvn -s maven_settings.xml --projects grpc -am -DskipTests=true package
-mvn -s maven_settings.xml --projects web -am -DskipTests=true package
-
-# For some reason, below command is what works locally - above commands cause mvn to loop (?)
-# mvn -s maven_settings.xml -am -DskipTests=true package
+mvn -s maven_settings.xml clean package -DskipTests
```

Comment:
I was getting those 401 errors when trying to run this script. We fixed it by running `mvn clean package -DskipTests`, then trying the script again. I figured we can just switch the script over to use the more general `mvn clean package`, since that seems to be more reliable and is just as fast as running the two individual commands.